content/en/tracing/trace_collection/library_config/nodejs.md

```diff
@@ -26,87 +26,211 @@ Tracer settings can be configured with the following environment variables:
 Enable debug logging in the tracer.
 
+`DD_TRACING_ENABLED`
+: **Configuration**: N/A<br>
+**Default**: `true`<br>
+Whether to enable tracing.
+
 `DD_TRACE_AGENT_URL`
-: **Default**: `http://localhost:8126`<br>
+: **Configuration**: `url`<br>
+**Default**: `http://localhost:8126`<br>
 The URL of the Trace Agent that the tracer submits to. Takes priority over hostname and port, if set. Supports Unix Domain Sockets in combination with the `apm_config.receiver_socket` in your `datadog.yaml` file, or the `DD_APM_RECEIVER_SOCKET` environment variable.
 
 `DD_TRACE_AGENT_HOSTNAME`
-: **Default**: `localhost`<br>
+: **Configuration**: `hostname`<br>
+**Default**: `localhost`<br>
 The address of the Agent that the tracer submits to.
 
 `DD_TRACE_AGENT_PORT`
-: **Default**: `8126`<br>
+: **Configuration**: `port`<br>
+**Default**: `8126`<br>
 The port of the Trace Agent that the tracer submits to.
 
 `DD_DOGSTATSD_PORT`
-: **Default**: `8125`<br>
+: **Configuration**: `dogstatsd.port`<br>
+**Default**: `8125`<br>
 The port of the DogStatsD Agent that metrics are submitted to.
 
 `DD_LOGS_INJECTION`
-: **Default**: `false`<br>
+: **Configuration**: `logInjection`<br>
+**Default**: `false`<br>
 Enable automatic injection of trace IDs in logs for supported logging libraries.
 
 `DD_TRACE_SAMPLE_RATE`
-: Percentage of spans to sample as a float between `0` and `1`. Defaults to the rates returned by the Datadog Agent.
+: **Configuration**: `sampleRate`<br>
+**Default**: Defaults to deferring the decision to the agent.<br>
+Controls the ingestion sample rate (between 0.0 and 1.0) between the agent and the backend.
 
 `DD_TRACE_RATE_LIMIT`
-: Percentage of spans to sample as a float between `0` and `1`. Defaults to `100` when `DD_TRACE_SAMPLE_RATE` is set. Otherwise, delegates rate limiting to the Datadog Agent.
+: **Configuration**: `rateLimit`<br>
+**Default**: Defaults to `1.0` when `DD_TRACE_SAMPLE_RATE` is set. Otherwise, delegates rate limiting to the Datadog Agent.
+Ratio of spans to sample as a float between `0.0` and `1.0`. <br>
 
-`DD_TRACE_SAMPLING_RULES` 
-: A JSON array of objects. Each object must have a "sample_rate", and the "name" and "service" fields are optional. The "sample_rate" value must be between 0.0 and 1.0 (inclusive). Rules are applied in configured order to determine the trace's sample rate. If omitted, the tracer defers to the Agent to dynamically adjust sample rate across all traces.
+`DD_TRACE_SAMPLING_RULES`
+: **Configuration**: `samplingRules`<br>
+**Default**: `[]`<br>
+Sampling rules to apply to priority sampling. A JSON array of objects. Each object must have a `sample_rate` value between 0.0 and 1.0 (inclusive). Each rule has an optional `name` and `service` fields, which are regexes to match against a trace's `service` and `name`. Rules are applied in configured order to determine the trace's sample rate. If omitted, the tracer defers to the Agent to dynamically adjust sample rate across all traces.
```

Comment:
```suggestion
Sampling rules to apply to priority sampling. A JSON array of objects. Each object must have a `sample_rate` value between 0.0 and 1.0 (inclusive). Each rule has optional `name` and `service` fields, which are regexes to match against a trace's `service` and `name`. Rules are applied in configured order to determine the trace's sample rate. If omitted, the tracer defers to the Agent to dynamically adjust sample rate across all traces.
```

Also, name and service are regex _strings_ aren't they? Do we support actual RegExp instances? Should maybe clarify that.

Reply:
Yeah, looks like strings